### DISCUSSION TITLE
Switch User option not showing

### DISCUSSION THREAD

**Discussion by Cardi619** [lxqt/lxqt#2424] [Q&A]
Hi,

I'm very happy with the way Lubuntu 22.04 (=lxqt) put new use to my old laptops for my kids to use. For practical reasons I would like them to use 1 computer in my home office. Unfortunately I can't seem to find the "Switch User" option behind the Leave menu item.

I've done some research and Lubuntu 22.04 (LXQT), Lubuntu 23.04 (LXQT), Debian LXQT all seem to lack this option.
Other desktop environments Xubuntu (XFCE), Debian LXDE, Lubuntu 18.04 (LXDE) all seem to feature this option.

So can I conclude the lack of this feature is related to LXQT? Or is there an easy way to enable it?

Thanks in advance!

Cheers,
Luc

**Comment by stefonarch** [comment]
Basically we are still missing a "save session" functionality, among the rest needed for switching sessions.

**Reply by brush** [reply]
i just want to make explicit that for me, and numerous other people in a variety of forums, this is a dealbreaker for using LXQT in a shared environment, forcing me/us off of lubuntu and into other distros. ideally this would help prioritize it as a bugfix.

**Reply by stefonarch** [reply]
On x11 this needs quite a lot of work to do as open windows and files need to be saved in a session manager.

But multiple user sessions are supported natively on wayland and especially by kwin_wayland (logout and select "Switch user").  For other wayland sessions like labwc or wayfire It is possible to login on tty and use one of the[ startlxqt*](https://github.com/stefonarch/LXQt-Wayland-files/tree/main/start_scripts) scripts. Not sure if Lubuntu 24.10 will meet the requirements for LXQt 2.0 though.

https://github.com/stefonarch/lxqt-kwin-session
https://github.com/stefonarch/LXQt-Wayland-files

**Comment by stefonarch** [comment]
On x11 this needs quite a lot of work to do as open windows and files need to be saved in a session manager.

But multiple user sessions are supported natively on wayland and especially by kwin_wayland (logout and select "Switch user").  For other wayland sessions like labwc or wayfire It is possible to login on tty and use one of the[ startlxqt*](https://github.com/stefonarch/LXQt-Wayland-files/tree/main/start_scripts) scripts. Not sure if Lubuntu 24.10 will meet the requirements for LXQt 2.0 though.

https://github.com/stefonarch/lxqt-kwin-session
https://github.com/stefonarch/LXQt-Wayland-files

**Comment by xavierog** [comment]
> Or is there an easy way to enable it?

If I understand your use case correctly, you want to spawn a second desktop session so your kids can mess with your computer using a separate user account and without having to end your regular desktop session.

If so, try this command:
```
qdbus --system org.freedesktop.DisplayManager /org/freedesktop/DisplayManager/Seat0 org.freedesktop.DisplayManager.Seat.SwitchToGreeter
```

It instructs SDDM to show a "greeter" (i.e. login/password user interface) on the first available vt (virtual terminal). That makes it possible to open a second desktop session with another user and/or another desktop environment. It also works around the need to save the entire session (described by stefonarch) as sessions end up running in parallel. One downside is that two users running desktop sessions on the same machine have to share hardware and thus it may be necessary to deal with e.g. pulseaudio subtleties.
